utils/summon.cfg

```diff
@@ -51,6 +51,12 @@ For more information, choose the 'Summoning Help' rightclick option which appear
 #enddef
 
 #define SUMMON_MENU_ITEM ID DESCRIPTION COST SUMMONER UNIT
+    #arg EXCEPT_FROM
+    C*,K*
```

Comment:
you probably should handle overlays as well, *^C* style

Reply:
Like this: ``C*,K*,*^C*,*^K*`` ?